src/org/usfirst/frc4904/autonly/RightSideTime.java

```diff
@@ -1,9 +1,18 @@
 package org.usfirst.frc4904.autonly;
 
 import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.Noop;
 
 public class RightSideTime extends SideSwitchTime {
 	public RightSideTime () {
-		super(RobotMap.gamefield.ourSwitch.right);
+		super();
```

Comment:
Why does this call super and differ from the rest of the SideSwitchTime-based commands?